h2o-algos/src/main/java/hex/tree/SharedTree.java

```diff
@@ -196,6 +197,11 @@ protected void ignoreInvalidColumns(int npredictors, boolean expensive) {
     if (_parms._build_tree_one_node) {
       warn("_build_tree_one_node", "Single-node tree building is not supported in this version of H2O.");
     }
+    if (!StringUtils.isNullOrEmpty(_parms._in_training_checkpoints_dir)) {
```

Comment:
what happens exactly if it's empty string? is it saved in a default dir?
when looking at `GBM.java` below, it looks like it will save all models to the root… not sure that's what we want.

Reply:
Good point, its not what we want, I would rather do nothing like export_checkpoints_dir = ""